src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDateTime by;
+    private static final DateTimeFormatter OUTPUT_DATE_FORMAT = DateTimeFormatter.ofPattern("MMM d yyyy, hh:mm a");;
+
+    public Deadline(String description, LocalDateTime by) {
+        super(description);
+        this.by = by;
+    }
+
+    public LocalDateTime getBy() {
+        return by;
```

Comment:
The method name getBy may not be as intuitive to other people reading your code. Perhaps you could change the field name to deadline and method name to getDeadline instead?